spec/koans.js

```diff
@@ -39,50 +47,57 @@ describe('`const` is like `let` plus read-only. ', () => {
   describe('scalar values are read-only', () => {
     it('number are read-only', () => {
       const constNum = 0;
-      constNum = 1;
+      //constNum = 1; cannot reassign a constat !
 
-      //expect(constNum).toBe(0);
+      expect(constNum).toBe(0);
     });
 
     it('string are read-only', () => {
       const constString = "I am a const";
-      constString = "Cant change you?";
+      //constString = "Cant change you?"; cannot reassign a constat !
 
-      //expect(constString).toBe("I am a const");
+      expect(constString).toBe("I am a const");
     });
 
   });
+
   
-  /*var, let or const? notChangeable = 23;*/
 
+  const notChangeable = 23;
   it('const scope leaks too', () => {
-    //expect(notChangeable).toBe(23);
+    expect(notChangeable).toBe(23);
   });
-  
+
+  // ######################### NO IDEA WHAT DOES THIS WANT ##############################//
+
   describe('complex types are NOT fully read-only', () => {
 
     it('arrays is not fully read-only', () => {
       const arr = [42, 23];
 
-      //expect(arr[0]).toBe(0);
+      expect(arr[0]).toBe(0);
```

Comment:
Here to demonstrate that value inside the array can be changed you could add something like ```arr[0] = 0;```